app/src/main/java/com/asfoundation/wallet/manage_wallets/RemoveWalletFragment.kt

```diff
@@ -231,12 +231,16 @@ class RemoveWalletFragment : BasePageViewFragment() {
     Card(
       colors = CardDefaults.cardColors(containerColor = WalletColors.styleguide_blue),
       border = BorderStroke(1.dp, styleguide_pink),
-      modifier = Modifier.padding(horizontal = 16.dp)
+      modifier = Modifier
```

Comment:
just a detail, the padding used for this AlertCard and the Buttons is showing up as larger than the previous card padding (BalanceCard)

Reply:
it was design to be like this 
https://xd.adobe.com/view/d5d0e051-89c6-4c75-930c-772e1a178286-8083/screen/581450c9-6768-469b-905e-c7dbd0916e73/specs/